app/pb_kits/playbook/pb_progress_step/_progress_step.jsx

```diff
@@ -6,13 +6,13 @@ import { buildCss } from '../utilities/props'
 import { globalProps } from '../utilities/globalProps.js'
 
 type ProgressStepProps = {
-  className?: string,
-  data?: string,
-  id?: string,
+  className?: String,
```

Comment:
I think we're aiming to keep these lowercase. https://github.com/powerhome/playbook/pull/969/files